Source/Core/Core/PowerPC/JitArm64/JitArm64_Branch.cpp

```diff
@@ -80,13 +80,6 @@ void JitArm64::WriteBranchWatch(u32 origin, u32 destination, UGeckoInstruction i
 {
   const ARM64Reg branch_watch = EncodeRegTo64(reg_a);
   MOVP2R(branch_watch, &m_branch_watch);
-  LDRB(IndexType::Unsigned, reg_b, branch_watch, Core::BranchWatch::GetOffsetOfRecordingActive());
-  FixupBranch branch_over = CBZ(reg_b);
-
-  FixupBranch branch_in = B();
-  SwitchToFarCode();
-  SetJumpTarget(branch_in);
-
   const ARM64Reg float_emit_tmp = EncodeRegTo64(reg_b);
   ABI_PushRegisters(gpr_caller_save);
   m_float_emit.ABI_PushRegisters(fpr_caller_save, float_emit_tmp);
```

Comment:
You can replace `float_emit_tmp` with `ARM64Reg::X8`. (This is safe because the previous line saved all caller-saved GPRs, which includes X8.) This will let you get rid of the `reg_b` parameter.